Zend/zend_vm_def.h

```diff
@@ -8935,7 +8935,29 @@ ZEND_VM_COLD_CONST_HANDLER(197, ZEND_MATCH_ERROR, CONST|TMPVARCV, UNUSED)
 
 	SAVE_OPLINE();
 	op = GET_OP1_ZVAL_PTR_UNDEF(BP_VAR_R);
-	zend_throw_exception_ex(zend_ce_unhandled_match_error, 0, "Unhandled match value of type %s", zend_zval_type_name(op));
+
+	// For simpler types where there is a convenient stringified version, include that
+	// in the error message. Otherwise just show its type.
+	switch (Z_TYPE_P(op)) {
```

Comment:
Please indent the code block properly

Reply:
This is the coding style used in the switch block I was copying from: https://heap.space/xref/php-src/ext/standard/var.c?r=ae8647d9#291

I'm good with whatever, but that does seem to be the "proper" style.  (Or at least a proper style.)

Reply:
That should have been fixed also, but not as part of this PR. Most uses I see in PHP source indent `case` statement without braces. See https://heap.space/xref/php-src/ext/standard/var.c?r=ae8647d9#104 or https://heap.space/xref/php-src/ext/standard/var.c?r=ae8647d9#1023

Reply:
Good ol' php-src... :sweat_smile: OK, will update.